src/main/scala/sbtunidoc/BaseUnidocPlugin.scala

```diff
@@ -0,0 +1,50 @@
+package sbtunidoc
+
+import sbt.Keys._
+import sbt._
+
+object BaseUnidocPlugin extends AutoPlugin {
+  object autoImport extends UnidocKeys
+  import autoImport._
+
+  override def projectSettings = Seq(
+    unidoc in Compile := Seq.empty,
+    unidoc in Test := Seq.empty
+  )
+
+  def commonSettings(sc: Configuration): Seq[sbt.Def.Setting[_]] = Seq(
```

Comment:
I normally use the name `commonSettings` specifically for common settings in `build.sbt`. I think it's better to call this baseUniDocSettings, or something along the line.